final_task/rss_parser/rss_parser.py

```diff
@@ -0,0 +1,156 @@
+import json
+import feedparser
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass
+class ArgParser:
+    """class for reading console arguments"""
+    source: str = ""
+    status_version: bool = False
+    status_json: bool = False
+    status_verbose: bool = False
+    limit: int = 1
+
+    arg_parser = argparse.ArgumentParser(description="Pure Python command-line RSS reader.")
+    arg_parser.add_argument("source", type=str, help="RSS URL")
+    arg_parser.add_argument("--version", action="store_true", help="Print version info")
+    arg_parser.add_argument("--json", action="store_true", help=" Print result as JSON in stdout")
+    arg_parser.add_argument("--verbose", action="store_true", help="Outputs verbose status messages")
+    arg_parser.add_argument("--limit", type=int, default=1, help="Limit news topics if this parameter provided")
+    arg_parser_args = arg_parser.parse_args()
+
+    def __post_init__(self):
+        self.source = self.arg_parser_args.source
+        self.status_version = self.arg_parser_args.version
+        self.status_json = self.arg_parser_args.json
+        self.status_verbose = self.arg_parser_args.verbose
+        self.limit = self.arg_parser_args.limit
```

Comment:
You don't really need separate class for this. You will have all parsed arguments already after line arg_parser_args = arg_parser.parse_args()